src/cmd/linuxkit/Makefile

```diff
@@ -6,6 +6,7 @@ export GO_FLAGS=-mod=vendor
 ifeq ($(OS),Windows_NT)
 LINUXKIT?=bin/linuxkit.exe
 GOOS?=windows
+SUFFIX?=.exe
```

Comment:
I do not understand this. As `LINUXKIT?=bin/linuxkit.exe`, what do we need the `SUFFIX` for? The change to the last line to make it `-o $(LINUXKIT)$(SUFFIX)` would make it `linuxkit.exe.exe`, wouldn't it?

Reply:
`go build -o linuxkit$(SUFFIX)` will produce `linuxkit` on everything except windows where it will be `linuxkit.exe`
Therefore when we `tar -C bin -xf tmp.tar` on linux we create `bin/linuxkit` and on windows we create `bin/linuxkit.exe`

I just realised I can achieve the same result with `$(notdir $(LINUXKIT))` so I'll do that instead to save the extra variable